index.html

```diff
@@ -0,0 +1,174 @@
+
+                    <div class="service-block">
+                        <div class="service-logo">
+                            <img src="/img/services-item-6.svg" alt="Feel Like A Star">
+                        </div>
+                        <div class="service-text">
+                            <h3><a href="#" class="service-text-link">Feel Like A Star</a></h3>
+                            <p class="about-service">Quisque hendrerit purus dapibus, ornare nibh vitae, viverra nibh.
+                                Fusce vitae aliquam tellus.</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section class="portfolio">
+            <h2 class="portfolio-text">Portfolio</h2>
+            <p class="portfolio-description">Duis mollis, est non commodo luctus,
+                nisi erat porttitor ligula, eget lacinia odio sem nec elit.</p>
+            <div class="tag-field">
+                <a class="tag-links" href="#"><span class="tag tag-active">All</span></a>
+                <a class="tag-links" href="#"><span class="tag">Web Disignx</span></a>
+                <a class="tag-links" href="#"><span class="tag">Graphic Design</span></a>
+                <a class="tag-links" href="#"><span class="tag">Artwork</span></a>
+            </div>
+
+            <div class="portfolio-boxes">
+                <img src="img/portfolio/portfolio-1.svg" alt="photo 1" class="portfolio-photo">
+                <img src="img/portfolio/portfolio-2.svg" alt="photo 2" class="portfolio-photo">
+                <img src="img/portfolio/portfolio-3.svg" alt="photo 3" class="portfolio-photo">
+                <img src="img/portfolio/portfolio-4.svg" alt="photo 4" class="portfolio-photo">
+                <img src="img/portfolio/portfolio-5.svg" alt="photo 5" class="portfolio-photo">
+                <img src="img/portfolio/portfolio-6.svg" alt="photo 6" class="portfolio-photo">
+                <img src="img/portfolio/portfolio-7.svg" alt="photo 7" class="portfolio-photo">
+                <img src="img/portfolio/portfolio-8.svg" alt="photo 8" class="portfolio-photo">
+                <img src="img/portfolio/portfolio-9.svg" alt="photo 9" class="portfolio-photo">
+                <img src="img/portfolio/portfolio-10.svg" alt="photo 10" class="portfolio-photo">
+                <img src="img/portfolio/portfolio-11.svg" alt="photo 11" class="portfolio-photo">
+                <img src="img/portfolio/portfolio-12.svg" alt="photo 12" class="portfolio-photo">
+                <img src="img/portfolio/portfolio-1.svg" alt="photo 1" class="portfolio-photo">
+
+            </div>
+        </section>
+
+        <div class="last-border"></div>
+    </main>
+
+
+    <footer>
+        <span class="copyright">© Copyright 2013</span>
+        <div class="social">
+            <a href="#" class="social-button"><span><img src="/img/ico-facebook.svg" alt="facebook"></span></a>
```

Comment:
The same as with back and forward arrows, you can move these icons to css as background-image